sycl/unittests/scheduler/InOrderQueueHostTaskDeps.cpp

```diff
@@ -40,7 +40,8 @@ TEST_F(SchedulerTest, InOrderQueueHostTaskDeps) {
 
   context Ctx{Plt};
   queue InOrderQueue{Ctx, default_selector_v, property::queue::in_order()};
-
+  if (!InOrderQueue.get_device().has(aspect::usm_shared_allocations))
```

Comment:
Why do we need any changes in unit-tests? Those are run on mock devices and we can just make mock device support necessary APIs to simplify things.

Reply:
Ok, was not aware of that, thanks!